--- a/src/common/subsys_types.h
+++ b/src/common/subsys_types.h
@@ -17,6 +17,7 @@
 
 #include <algorithm>
 #include <array>
+#include <cstdint>
 
 enum ceph_subsys_id_t {
   ceph_subsys_,   // default
--- a/src/rocksdb/db/compaction/compaction_iterator.h
+++ b/src/rocksdb/db/compaction/compaction_iterator.h
@@ -9,6 +9,7 @@
 #include <string>
 #include <unordered_set>
 #include <vector>
+#include <cstdint>
 
 #include "db/compaction/compaction.h"
 #include "db/compaction/compaction_iteration_stats.h"
--- a/src/common/Cycles.h
+++ b/src/common/Cycles.h
@@ -32,6 +32,8 @@
 #ifndef CEPH_CYCLES_H
 #define CEPH_CYCLES_H
 
+#include <cstdint>
+
 /**
  * This class provides static methods that read the fine-grain CPU
  * cycle counter and translate between cycle-level times and absolute
--- a/src/rocksdb/include/rocksdb/rocksdb_namespace.h
+++ b/src/rocksdb/include/rocksdb/rocksdb_namespace.h
@@ -8,3 +8,5 @@
 #ifndef ROCKSDB_NAMESPACE
 #define ROCKSDB_NAMESPACE rocksdb
 #endif
+
+#include <cstdint>
--- a/src/rocksdb/third-party/folly/folly/synchronization/detail/ProxyLockable-inl.h
+++ b/src/rocksdb/third-party/folly/folly/synchronization/detail/ProxyLockable-inl.h
@@ -14,6 +14,7 @@
 #include <mutex>
 #include <stdexcept>
 #include <utility>
+#include <system_error>
 
 namespace folly {
 namespace detail {
--- a/src/librbd/api/PoolMetadata.h
+++ b/src/librbd/api/PoolMetadata.h
@@ -9,6 +9,7 @@
 
 #include <map>
 #include <string>
+#include <cstdint>
 
 namespace librbd {
 
